components/mapbox/actions/generate-directions/generate-directions.mjs

```
@@ -0,0 +1,60 @@
+      ],
+    },
+  },
+  async run({ $ }) {
+    const directions = await this.mapbox.getDirections({
+      $,
+      transportationMode: this.transportationMode,
+      coordinates: `${this.startCoordinate};${this.endCoordinate}`,
+      params: {
+        steps: this.steps,
+        alternatives: this.alternatives,
+        exclude: this.exclude && this.exclude.join(","),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)